app/sections/view-solution-capabilities/macro.njk

```diff
@@ -3,12 +3,19 @@
 
 {% macro viewSolutionCapabilities(params) %}
   {% if params.section %}
-    <div data-test-id="view-solution-capabilities">
-      {{ viewSectionHeading({text: 'Capabilities met'}) }}
-      <div class="nhsuk-u-padding-left-3">
-        {{ viewQuestionDataBulletlist({
-          questionData: params.section.answers['capabilities-met']
-        }) }}
+    <div class="nhsuk-width-container" data-test-id="view-solution-capabilities">
+      <div class="nhsuk-grid-row">
+        {{ viewSectionHeading({text: 'Capabilities met'}) }}
+          {% if params.section.description %}
+            <p class="nhsuk-grid-column-two-thirds">

```

Comment:
Could we create a div around both components that includes the "nhsuk-grid-column-two-thirds" class. That way we don't need to duplicate it